poc/package.json

```diff
@@ -24,6 +24,8 @@
     "@angular/router": "18.1.1",
     "angular-resizable-element-labworks": "7.0.3",
     "d3": "^5.8.0",
+    "dom-to-image": "^2.6.0",
```

Comment:
Why do we need this library?

Reply:
Since we just do a POC let's keep it for now and remove it later if not needed